src/firmware_new/boards/frankie_v1/main.c

```diff
@@ -129,17 +109,21 @@ int main(void)
 
     /* Initialize all configured peripherals */
     MX_GPIO_Init();
-    MX_ETH_Init();
     MX_USART3_UART_Init();
     MX_USB_OTG_FS_PCD_Init();
+    MX_CRC_Init();
+    MX_LWIP_Init();
     /* USER CODE BEGIN 2 */
+    HAL_Delay(1000);
```

Comment:
Why do we need this delay?

Reply:
nuked it, was from earlier times when I was still debugging ethernet